proposed/http-client/http-client-meta.md

```diff
@@ -3,8 +3,12 @@ HTTP Client Meta Document
 
 ## Summary
 
+HTTP request and responses are the two fundamental objects in web programming. All clients communicating to an external API is using some form of HTTP client. Many libraries are coupled to one specific client or is implementing a client themselves. This leads to bad library design, version conflicts or too much code note related to the library domain. 
```

Comment:
Many libraries are coupled to one specific client or implement a client and/or adapter layer themselves.